homeworks/homework-vue/src/models/news-item.model.ts

```diff
@@ -15,7 +15,11 @@ export class NewsItemModel {
     this.votes = votes;
   }
 
-  compareTo(other: NewsItemModel) : number {
-    return other.votes - this.votes
+  compareToDescending(other: NewsItemModel): number {
+    return other.votes - this.votes;
```

Comment:
```suggestion
    return (-1) * this.compareToAscending(other)
```
:thinking: maybe? Or just keep your code, this idea is actually a little less obvious...